### PR TITLE
adds support for X-PUBLISHED-TTL property.

### PR DESCRIPTION
**Pull request by stopfstedt**
Hey there, 

I'm currently working on upgrading from version `0.16.1` and noticed that the ability to set this property did not make it into v2 yet.

I'm hereby humbly requesting that this functionality gets added back in, please see attached changes. 

Thanks!

### CODE REVIEW

**Comment by markuspoerschke** [comment]
Hey @stopfstedt thanks for your contribution!

From the code perspective, the changes are looking good, but the documentation is missing. Can you add some explanatory words to the documentation describing what this property does?

**Comment by stopfstedt** [comment]
Hi @markuspoerschke, I added a new sub-page to the docs, in the Advanced section. 

image

File: website/docs/non-standard-properties.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+The [`X-PUBLISHED-TTL`](http://msdn.microsoft.com/en-us/library/ee178699(v=exchg.80).aspx) property specifies a suggested iCalendar file download frequency for clients and servers with sync capabilities.
+
+Use the `setPublishedTTL()` method with a [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Durations)-formatted string to set this duration.
+
+```php
+use Eluceo\iCal\Domain\Entity\Calendar;
+
+$calendar = new Calendar();
+// set the duration to 2 hours
+$calendar->setPublishedTTL('PT2H')
```
Comment (markuspoerschke) [inline website/docs/non-standard-properties.md]: The method `setPublishedTTL` expects an instance of `\DateInterval`, so I think, the example must be changed to this, or?

```suggestion
$calendar->setPublishedTTL(new DateInterval('PT2H'))
```

**Review by markuspoerschke — CHANGES_REQUESTED** [review]

File: website/docs/non-standard-properties.md (hunk 1 above)
Comment (stopfstedt) [reply website/docs/non-standard-properties.md]: doh! of course. thanks for catching that.

**Review by markuspoerschke — APPROVED** [review]

**Pull request merged**